TCAT/Utilities/SearchTableViewHelpers.swift

```diff
@@ -102,9 +102,10 @@ func sectionIndexesForBusStop() -> [String: Int] {
 func parseGoogleJSON(searchText: String, json: JSON) -> Section {
     var itemTypes: [ItemType] = []
     let filteredBusStops = getAllBusStops().filter({(item: BusStop) -> Bool in
-        let stringMatch = item.name.lowercased().range(of: searchText.lowercased())
-        return stringMatch != nil
+        let levenshteinScore = String.fuzzPartialRatio(str1: item.name.lowercased(), str2: searchText.lowercased())
+        return levenshteinScore > 75
```

Comment:
create a `constant` variable to abstract magic number from code, possible as variable in struct or enclosing class